var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -55,6 +55,7 @@ class Paraview(CMakePackage, CudaPackage):
             values=('native', 'fermi', 'kepler', 'maxwell',
                     'pascal', 'volta', 'turing', 'ampere', 'all', 'none'),
             description='CUDA architecture')
+    variant('debug', default=False, description="Enable debug flags")
```

Comment:
Since this is deriving from `CMakePackage`, there is a `build_type` variant already registered in the package :-)
(See: `spack info paraview`)

Reply:
I know this build-type variant, but the first idea was to regroup flags which are not included in. 

I'm not a paraview maintainer so I don't know if this flag should be added in the build-type or if it's a bad idea. @vicentebolea @danlipsa Do you think its a better idea to add the VTK_DEBUG_LEAKS flag in the debug build_type ?
If not, maybe rename this variant "advanced_debug" for additional flags ?